prow/daily-e2e-rbac-no_auth-skew.sh

```diff
@@ -31,4 +31,4 @@ WD=$(cd $WD; pwd)
 ROOT=$(dirname $WD)
 
 export E2E_TARGET="e2e_version_skew"
-${ROOT}/prow/e2e-suite.sh
+${ROOT}/prow/e2e-suite.sh --cluster_wide --test_vm
```

Comment:
The skew test is a separate target that does not test the vm for now so we could remove all `--test_vm` on *-skew.sh

Reply:
Done.